jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/resources/LambdaResources.kt

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation.resources
+
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient
+import software.amazon.awssdk.services.cloudformation.model.StackResourceSummary
+import software.amazon.awssdk.services.cloudformation.model.StackSummary
+import software.aws.toolkits.jetbrains.core.CachedResource
+import software.aws.toolkits.jetbrains.core.CachedResourceBase
+
+object CloudFormationResources {
+    private val listStacks: CachedResource<List<StackSummary>> by lazy {
+        CloudFormationCachedResource { listStacksPaginator().stackSummaries().toList() }
+    }
+
+    fun listStacks() = listStacks
```

Comment:
Any reason not to just expose this as a `LIST_STACKS` constant val?

Reply:
Wanted some consistentcy, that's all

Reply:
Consistency with what? https://github.com/aws/aws-toolkit-jetbrains/blob/master/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/resources/LambdaResources.kt#L12

Reply:
Just within this class 😄 
Mixing consts and functions was all. I dont have strong opinion